Makefile

```diff
@@ -1,8 +1,22 @@
 #
 # Makefile for more convenient building of the Linode CLI and its baked content
 #
+PKG_PATH =
```

Comment:
I think it might be better to name it as `MOD_PATH` as well as other vars, because in Python, a sub-directory is called a module, not a package like what it's called in Golang.